Makefile

```diff
@@ -11,7 +11,7 @@ stop:
 
 .PHONY: tests
 tests:
-	docker-compose -f docker/docker-compose.test.yml run --rm --service-ports api_tests
+	docker-compose -f docker/docker-compose.test.yml run --rm --service-ports app-tests
```

Comment:
minor comment, I'd say separating words in a service/container name with `-` is good, as we assign special meaning to `_` (e.g. https://hub.docker.com/r/elifesciences/sciencebeam-gym_unstable which is a variant of the `sciencebeam-gym` image pushed on every commit)